cumulus_library/template_sql/utils.py

```diff
@@ -164,12 +164,12 @@ def _check_schema_if_exists(
         cursor.execute(query)
         schema_str = str(cursor.fetchone()[0])
         required_fields = [coding_element]
-        if allow_partial:
-            required_fields + ["code", "system", "display"]
+        if not allow_partial:
+            required_fields += ["code", "system", "display"]
```

Comment:
This was a real bug where `allow_partial` was being ignored. I think I fixed it correctly, and then I added some unit testing for it.